patroni/ctl.py

```diff
@@ -113,7 +116,7 @@ def ctl(ctx):
 
 def get_dcs(config, scope):
     dcs_config = config.get('dcs', {})
-    dcs_config[list(dcs_config.keys())[0]]['scope'] = scope
+    [dcs_config[k].setdefault('scope', scope) for k in ['zookeeper', 'etcd', 'consul'] if k in dcs_config]
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expression "[dcs_config[k].setdefault('scope', scope) for k in ['zookeeper', 'etcd', 'consul'] if k in dcs_config]" is assigned to nothing](https://www.codacy.com/app/cyberdemn/patroni/file/2447659967/issues/source?bid=3127594&fileBranchId=3247017#l119)